be/src/vec/exec/vjdbc_connector.cpp

```diff
@@ -648,6 +724,8 @@ Status JdbcConnector::_register_func_id(JNIEnv* env) {
 
     RETURN_IF_ERROR(register_id(_executor_clazz, "getBlock", JDBC_EXECUTOR_GET_BLOCK_SIGNATURE,
                                 _executor_get_blocks_id));
+    RETURN_IF_ERROR(register_id(_executor_clazz, "getBlock", JDBC_EXECUTOR_GET_BLOCK_NEW_SIGNATURE,
```

Comment:
`JDBC_EXECUTOR_GET_BLOCK_NEW_SIGNATURE`, may be we can use a more meaningful name?

Reply:
will change to JDBC_EXECUTOR_GET_BLOCK_WITH_TYPES_SIGNATURE, is that OK?